src/client/Microsoft.Identity.Client/Http/Retry/ImdsRetryPolicy.cs

```diff
@@ -0,0 +1,67 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Net;
+using System.Threading.Tasks;
+using Microsoft.Identity.Client.Core;
+
+namespace Microsoft.Identity.Client.Http.Retry
+{
+    internal class ImdsRetryPolicy : IRetryPolicy
```

Comment:
Add a comment stating the retry policy specific to IMDS and the link to IMDS docs if they are public

Reply:
I'll link to our internal spec

Reply:
done